cli/tests/pcluster/config/test_pcluster_config.py

```diff
@@ -70,7 +70,12 @@ def test_get_latest_alinux_ami_id(mocker, boto3_stubber, path, boto3_response, e
     [
         (
             # ResourceS3Bucket available
-            {"Scheduler": "slurm", "ResourcesS3Bucket": "valid_bucket"},
+            {
+                "Scheduler": "slurm",
+                "ResourcesS3Bucket": "valid_bucket",
+                "ArtifactS3RootDirectory": "valid_dir",
+                "RemoveBucketOnDeletion": "valid_setting",
```

Comment:
where is this valid_setting value coming from?

Reply:
You are right, the `RemoveBucketOnDeletion ` parameter is not needed for this test